pkg/phlaredb/head.go

```diff
@@ -847,23 +846,12 @@ func (h *Head) Series(ctx context.Context, req *connect.Request[ingestv1.SeriesR
 	return connect.NewResponse(response), nil
 }
 
-// Closes closes the head
-func (h *Head) Close() error {
-	close(h.stopCh)
-
-	var merr multierror.MultiError
-	for _, t := range h.tables {
-		merr.Add(t.Close())
-	}
-
-	h.wg.Wait()
-	return merr.Err()
-}
-
```

Comment:
I removed the method because it is impossible to use it correctly:
1. `Close` is supposed to be called after `Flush`, and tables are closed at `Flush`, therefore the call always return an error. It worked, because we always created an empty head right away after flush.
2. There is no case when head is just closed without flushing data.